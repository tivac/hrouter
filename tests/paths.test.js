"use strict";

const hrouter = require("../dist/hrouter.cjs.js");

describe("hrouter", () => {
    beforeEach(() => {
        hrouter._routes = Object.create(null);
    });

    it("should register routes", () => {
        hrouter("/fooga", () => {});

        expect(hrouter._routes).toMatchSnapshot();
    });
});
