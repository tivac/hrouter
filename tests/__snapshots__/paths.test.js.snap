// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hrouter should register routes 1`] = `
Object {
  "fooga": Object {
    "fn": [Function],
  },
}
`;
